plugins/imuxsock/imuxsock.c

```diff
@@ -1240,7 +1240,11 @@ CODESTARTbeginCnfLoad
 	pModConf->bAnnotateSysSock = 0;
 	pModConf->bParseTrusted = 0;
 	pModConf->bParseHost = UNSET;
-	pModConf->bUseSpecialParser = 1;
+	#if (defined(__FreeBSD__) && (__FreeBSD__ >= 1200000))
```

Comment:
This is not right. Either, you want to check if __FreeBSD__ is >= 12 (this is defined by the compiler), or you want __FreeBSD_version from <sys/param.h>. This can give more fine grained version check. In this case, the precise version to check against is 1200061 , which is when this changed went into what then became FreeBSD 12.0.

Reply:
thx - sticking with the compiler as I understand that all stable FreeBSD 12 have RFC5424 on the system log socket. Right?

Reply:
All stable FreeBSD 12 should have RFC5424, yes. The recommended way is generally to use `__FreeBSD_version`, since the value of __FreeBSD__ is dependent on the system you build on, not the system you target, as I understand it.